ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -894,6 +952,25 @@ func (r *RayJobReconciler) constructRayClusterForRayJob(rayJobInstance *rayv1.Ra
 	return rayCluster, nil
 }
 
+func (r *RayJobReconciler) getPreviousAndNextScheduleDistance(ctx context.Context, currentTime time.Time, rayJobInstance *rayv1.RayJob) (time.Duration, time.Duration, error) {
```

Comment:
I find it slightly confusing that this function is called `getPreviousAndNextScheduleDistance` but it returns (next, previous). Was this function also copied from k8s implementation?

Reply:
Yeah understood, I just mean you named the function `getPreviousAndNextScheduleDistance` but you actually return next and previous. The order of the name and returned items should be the same